src/main/java/com/jcraft/jsch/Slf4jLogger.java

```diff
@@ -1,48 +1,63 @@
 package com.jcraft.jsch;
 
+import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class Slf4jLogger implements com.jcraft.jsch.Logger {
 
-  private static final org.slf4j.Logger logger = LoggerFactory.getLogger(JSch.class);
+  private static final Logger stlogger = LoggerFactory.getLogger(JSch.class);
+  private final Logger logger;
 
-  public Slf4jLogger() {}
+  public Slf4jLogger() {
+    this(stlogger);
+  }
+
+  Slf4jLogger(Logger logger) {
+    this.logger = logger;
+  }
 
   @Override
   public boolean isEnabled(int level) {
-    switch (level) {
-      case com.jcraft.jsch.Logger.DEBUG:
-        return logger.isDebugEnabled();
-      case com.jcraft.jsch.Logger.INFO:
-        return logger.isInfoEnabled();
-      case com.jcraft.jsch.Logger.WARN:
-        return logger.isWarnEnabled();
-      case com.jcraft.jsch.Logger.ERROR:
-      case com.jcraft.jsch.Logger.FATAL:
-        return logger.isErrorEnabled();
-      default:
-        return logger.isTraceEnabled();
+    switch(level) {
+        case com.jcraft.jsch.Logger.DEBUG:
+            return logger.isDebugEnabled();
+        case com.jcraft.jsch.Logger.ERROR:
+        case com.jcraft.jsch.Logger.FATAL:
+            return logger.isErrorEnabled();
+        case com.jcraft.jsch.Logger.INFO:
+            return logger.isInfoEnabled();
+        case com.jcraft.jsch.Logger.WARN:
+            return logger.isWarnEnabled();
```

Comment:
The formatting for this no longer seems to follow [Google Java Style Guide's guidelines in section 4.8.4](https://google.github.io/styleguide/javaguide.html#s4.8.4-switch).
Can we return the formatting back, since I created this file following their style guide?

Reply:
Also, I had purposely ordered the levels in the switch statement in order by severity (lowest to highest), can we also restore that order?